app/api/callbacks.py

```diff
@@ -4,7 +4,7 @@
 from app.repositories.s3_enrolment_repo import S3EnrolmentRepo
 from app.requests.callback_requests import CallbackRequest
 from app.responses import SuccessType
-from app.use_cases.create_new_callback import CreateNewCallback
+from app.use_cases import create_new_callback as uc
```

Comment:
why do you think this approach is better?

Reply:
I made this change to mock the usecase. Otherwise, it wasn't possible to do it.